src/templates/core/functions/isBlob.hbs

```diff
@@ -1,13 +1,3 @@
 export const isBlob = (value: any): value is Blob => {
-	return (
-		value !== null &&
-		typeof value === 'object' &&
-		typeof value.type === 'string' &&
-		typeof value.stream === 'function' &&
-		typeof value.arrayBuffer === 'function' &&
-		typeof value.constructor === 'function' &&
-		typeof value.constructor.name === 'string' &&
-		/^(Blob|File)$/.test(value.constructor.name) &&
-		/^(Blob|File)$/.test(value[Symbol.toStringTag])
-	);
+	return value instanceof Blob;
```

Comment:
Possible to inline this call wherever it's used instead of having a separate function? It's so straightforward now that we don't need a wrapper function around it...